src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import java.io.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The Duke class represents a simple chatbot application that helps manage tasks.
+ */
 public class Duke {
+    private static List<Task> tasks = new ArrayList<>();
+    /**
+     * The main method to start the Duke chatbot.
+     *
+     * @param args Command-line arguments (unused).
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke wizzer = new Duke();
+        String logo = "Wiz";
+        String folderPath = "./data/";
+        String filePath = "./data/duke.txt";
+        try {
+            tasks = readFile(filePath, folderPath);
+        } catch (DukeException e) {
+            System.out.println(e);
+        }
+        System.out.println("Hello from " + logo +
+                "\nWhat can I do for you?\n");
+        System.out.println("--------------------------");
```

Comment:
Since you have quite a lot of dashed lines to print, will it be better to create a field for them to avoid complicating the code?